src/Elliptic/FirstOrderComputeTags.hpp

```diff
@@ -58,7 +58,7 @@ struct FirstOrderSourcesCompute
   using return_type = db::item_type<base>;
   template <typename... SourcesArgs>
   static void function(const gsl::not_null<return_type*> sources,
-                       const db::const_item_type<VariablesTag>& vars,
+                       const typename VariablesTag::type& vars,
```

Comment:
In a recent PR (I think #2306) we started replacing `const_item_type`s with `const auto&` and the pointers by `const auto` instead of getting the `Tag::type`s. We should probably decide what to do and then be consistent about it. What do you think?

Reply:
You can't use `auto` in a function parameter declaration, so to be consistent you will need to use `Tag::type`.

Reply:
Right, I picked the wrong line to comment on. See comment below